ui/app/components/actions-dropdown.hbs

```diff
@@ -0,0 +1,57 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
```

Comment:
A componentized dropdown/select box, which we use on the Job page, the Task page, and within the Actions flyout (within the context of a job or a task)